schedule/yast/minimal+base/minimal+base@yast-svirt-hyperv.yaml

```diff
@@ -10,41 +10,13 @@ vars:
   PATTERNS: base,enhanced_base
   YUI_REST_API: 1
 schedule:
-  - installation/bootloader_start
-  - installation/setup_libyui
-  - installation/access_beta_distribution
-  - installation/product_selection/install_SLES
-  - installation/licensing/accept_license
-  - installation/registration/register_via_scc
-  - installation/module_registration/skip_module_registration
-  - installation/add_on_product/skip_install_addons
-  - installation/system_role/accept_selected_role_text_mode
-  - installation/partitioning/accept_proposed_layout
-  - installation/clock_and_timezone/accept_timezone_configuration
-  - installation/authentication/use_same_password_for_root
-  - installation/authentication/default_user_simple_pwd
+  software:
```

Comment:
only `select_patterns` under `software`. Please, apply to the rest of the files.

Reply:
Changed.